### ISSUE TITLE
Also modifiy HTTPS vhosts for HTTP-01 in Apache

### ISSUE THREAD

**Issue by bmw** [certbot/certbot#5763]
See https://community.letsencrypt.org/t/fatal-error-during-cert-install-on-plesk/57067.

**Comment by rushilsrivastava** [comment]
Continuing to face this issue. Unclear what caused this issue in the first place, and what I can do to prevent it. It works sometimes, and others it doesn't. Issue persists in both Nginx (#5764) and Apache.

**Comment by wanderingexplorer** [comment]
I think I am having the same issue. I have sites with Let's Encrypt certs and then I have setup Cloudflare and changed the nameservers for the domains to Cloudflare. Now Let's encrypt auto-renew is failing. I'm sure this was working perfectly fine before. Any update on when this issue might be fixed?

Thank you for your time.

**Comment by alexzorin** [comment]
@bmw any chance of reconsidering?

It's been biting users recently.

As @wanderingexplorer mentioned, the Apache authenticator is not able to succeed because of the CDN delivering all requests to the origin over HTTPS. It's seems like a fairly hard to identify issue for users.

Recently two threads where Cloudflare usage led to this problem:
- https://community.letsencrypt.org/t/not-able-to-stop-using-tls-sni-01/86795/8
- https://community.letsencrypt.org/t/invalid-response-unauthorized/84818/4

Other CDNs have [similar modes of operation](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/using-https-cloudfront-to-custom-origin.html). I imagine that "secure origin" will only become more prevalent.

Is there some specific harm that comes about from modifying HTTPS virtual hosts?

**Comment by bmw** [comment]
We actually did this in our 0.31.0 release with the change to modify all virtual hosts containing the domain being validated although the release notes don't make that very clear.

**Comment by alexzorin** [comment]
You are right -  I was accidentally testing from 0.28. Thanks, that's great.

**Comment by rushilsrivastava** [comment]
Awesome, glad this was added.
